src/migrators/bundle.ts

```diff
@@ -85,20 +88,13 @@ export class BundleMigrator implements Migrator {
       throw new Error(`Invalid migration sha256, must match ${VALID_SHA256_EXP}`)
     }
 
-    const ts = Date.now()
-    const sql = `
-      PRAGMA defer_foreign_keys = ON;
-      BEGIN;
-        ${body};
-
-        INSERT INTO ${this.tableName}
-          ('name', 'sha256', 'applied_at')
-        VALUES
-          ('${name}', '${sha256}', '${ts}');
-      COMMIT;
-      PRAGMA defer_foreign_keys = OFF;
+    const applied =
+      `INSERT INTO ${this.tableName}
+        ('name', 'sha256', 'applied_at')
+      VALUES
+        ('${name}', '${sha256}', '${Date.now()}');
```

Comment:
Noting we're not using bind params here.

Reply:
thank you